docker-compose.python3.yml

```diff
@@ -0,0 +1,11 @@
+# docker-compose for Pontoon python3 development
+#
+# Note: Requires docker-compose 1.10+.
+version: "2"
+services:
+  # Pontoon python3 base image
+  webapp.python3:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.py3.base
+    image: local/pontoon_base
```

Comment:
@jotes I think this `docker-compose.python3.yml` file is not doing anything because there is no `Dockerfile.py3.base` file so what should we do now?

Reply:
@aniruddha2000 the first of all -> thanks for all your work!

I have some thoughts/questions:
* I can't find any reference to `docker-compose.python3.yml` . Do you want to use it in the Makefile? How it will affect  the targets?
* `docker-compose.yml` allows to parametrize the version of Python.

Reply:
@jotes 

>     * I can't find any reference to `docker-compose.python3.yml` . Do you want to use it in the Makefile? How it will affect  the targets?

Can you provide me some reference how i can `docker-compose.python3.yml` this file in the Makefile?

Reply:
Theoretically, you can create a target in the Makefile and just pass it there via `docker-compose -f`.
However, I think we don't need to have a new service for Python 3 if we will parametrize existing services (web app).